test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -15,7 +15,7 @@
 SMDEBUG_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p3.8xlarge", processor="gpu")
 SMDEBUG_EC2_CPU_INSTANCE_TYPE = get_ec2_instance_type(default="c4.8xlarge", processor="cpu")
 
-
+@pytest.mark.usefixtures("feature_smdebug_present")
```

Comment:
Please leave 2 lines of space before the start of a function.

```suggestion

@pytest.mark.usefixtures("feature_smdebug_present")
```